### DISCUSSION TITLE
Publishing the app to the play store

### DISCUSSION THREAD

**Discussion by CommanderRedYT** [OpenDTU-App/opendtu-react-native#137] [Announcements]
As per googles guidelines, I need 20 beta testers to enroll the app into the play store. If someone is up for helping me with that, feel free to contact me!

**Comment by Balled239** [comment]
Im Happy to Support,
Regards Daniel

**Reply by CommanderRedYT** [reply]
Thank you very much, feel free to write me on the email in my profile!

**Comment by CommanderRedYT** [comment]
Update on this: Google Play really does not support open source projects that are smaller.
I tried it for a month now to get things going, but they only respond with that the closed beta testing is not sufficient for production access. I believe that getting the app into the play store will not be possible thanks to googles policies.

**Comment by CommanderRedYT** [comment]
Update: We did it!
